packages/react/src/components/Heading/README.md

```diff
@@ -13,9 +13,9 @@ export default () => <Heading>Content</Heading>;
 #### `as`: oneOf(`HEADING_LEVELS`)
 
 ```jsx
-import { Heading, HEADING_LEVELS } from "@heetch/flamingo-react";
+import { Heading } from "@heetch/flamingo-react";
 
-export default () => <Heading as={HEADING_LEVELS.h3} />;
+export default () => <Heading as={3} />;
```

Comment:
I REALLY LIKE THIS. Having to use constants all the time is a bit of a pain 😄 passing a number is 🚀